post.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <title>Distributed Press Reader</title>
-<style>
-	@import url("./style.css")
-</style>
+<link rel="stylesheet" href="./style.css" />
+<div id="post-container">
+  <!-- https://staticpub.mauve.moe/helloworld.html -->
+  <!-- hyper://staticpub.mauve.moe/helloworld.html -->
+  <!-- ipfs://bafybeifslnipwp5uanmhkckokwuse7h5gfrrjzqq4jg5oxewxbzrdcdawu -->
+  <distributed-post url="ipns://staticpub.mauve.moe/helloworld.html">
```

Comment:
We should add a script that sets the URL based on the querystring parameter. It could be inline with the HTML in a script tag without needing a separate file.